helm-charts/0.25.4/charts/keystone/templates/Identity.yaml

```diff
@@ -8,20 +8,20 @@
 # >/'  SPDX-License-Identifier: BSD-2-Clause
 # */
 
+{{- if .Values.global.useClusterSpiffeIds }}
+
 apiVersion: spire.spiffe.io/v1alpha1
 kind: ClusterSPIFFEID
 metadata:
   name: {{ include "keystone.fullname" . }}
 spec:
-  spiffeIDTemplate: "spiffe://{{ .Values.global.spire.trustDomain }}\
-    /workload/{{ include "keystone.fullname" . }}\
-    /ns/{{`{{ .PodMeta.Namespace }}`}}\
-    /sa/{{`{{ .PodSpec.ServiceAccountName }}`}}\
-    /n/{{`{{ .PodMeta.Name }}`}}"
+  spiffeIDTemplate: {{ .Values.global.vsecm.keystoneSpiffeIdTemplate }}
```

Comment:
spiffeIDTemplates come from the global values too right now.

This is again useful for cases where all of these need to be configured to follow a certain pattern where it does not alight with what VSecM uses by default (i.e.
instead of `domain/workload/etc/etc` `domain/n/$namespace/s/$serviceAccount` (similar to how Istio assigns spiffe ids)